src/build-scripts/build_pybind11.bash

```diff
@@ -46,8 +46,9 @@ if [[ -z $DEP_DOWNLOAD_ONLY ]]; then
     time cmake -S . -B ${PYBIND11_BUILD_DIR} -DCMAKE_BUILD_TYPE=Release \
                -DCMAKE_INSTALL_PREFIX=${PYBIND11_INSTALL_DIR} \
                -DPYBIND11_TEST=OFF \
-               ${PYBIND11_BUILD_OPTS}
-    time cmake --build ${PYBIND11_BUILD_DIR} --config Release --target install
+               -DCMAKE_MODULE_PATH=${CONAN_CMAKE_FILES} \
+               ${PYBIND11_BUILD_OPTS} ..
+    time cmake --build . --config Release --target install
```

Comment:
Again here, this script seems to have lost the ability to put the build dir anywhere we want.

Reply:
Fixed